apps/mull-ui/src/app/components/event-card/event-card.tsx

```diff
@@ -0,0 +1,58 @@
+import { IEvent } from '@mull/types';
+import React, { useState } from 'react';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faShareAlt, faCheck, faSignInAlt } from '@fortawesome/free-solid-svg-icons';
+
+import { formatDate } from '../../../utilities';
+
+import './event-card.scss';
+
+/* eslint-disable-next-line */
+export interface EventCardProps {
+  event: Partial<IEvent>;
+  style?: React.CSSProperties;
+}
+
+export const EventCard = ({ event, style = {} }: EventCardProps) => {
+  // TODO; set joined based on if current user is part of event
+  const [joined, setJoined] = useState<boolean>(false);
+
+  const { day, month, time } = formatDate(event.startDate);
+  return (
+    <div className="event-card-container" style={style}>
+      <img
+        className="event-card-image"
+        // TODO: Replace placeholder
+        src="https://www.citywindsor.ca/residents/parksandforestry/City-Parks/PublishingImages/Assumption%20Park%20Street%20View.JPG"
+        alt="Event"
+      />
+      <div className="event-card-datetime" data-testid="event-card-datetime">
+        <div>{`${month} ${day}`}</div>
+        <div>{time}</div>
+      </div>
+      {/* TODO: Add/remove user to event on press */}
+      <button
+        onClick={() => setJoined(!joined)}
+        className={`event-card-join ${joined ? 'joined' : ''}`}
+      >
+        <FontAwesomeIcon icon={joined ? faCheck : faSignInAlt} />
+      </button>
+      <div className="event-card-description">
+        <div className="event-card-text">
+          <div className="event-card-title">{event.title}</div>
+          {/* TODO: Replace placeholder distance*/}
+          <div className="event-card-location">14km • {event.location.point}</div>
```

Comment:
Shouldn't the distance also be dynamic?

Reply:
it is , but currently I have no way of calculating it

Reply:
I'll make distance a variable which is interpolated in. And then leave a comment mentionning that it should be implemented.

Reply:
We could do what bootstrap does. They have a generic `container` class with default padding. Then you can use `container fluid` if you'd like the container to take up the full screen.

Reply:
True, it's fine like this for now.